notes/condition.md

```diff
@@ -53,23 +48,113 @@ where we used $$\|{\bf A}\boldsymbol{x}\| \le \|{\bf A}\| \|\boldsymbol{x}\|, \f
 
 Then
 
-$$\frac{\|\Delta \boldsymbol{x}\|}{\|\boldsymbol{x}\|} \le \text{cond}({\bf A})\frac{\|\Delta \boldsymbol{b}\|}{\|\boldsymbol{b}\|}  \qquad (1)$$
+$$\frac{\|\Delta \boldsymbol{x}\|}{\|\boldsymbol{x}\|} \le \text{cond}({\bf A})\frac{\|\Delta \boldsymbol{b}\|}{\|\boldsymbol{b}\|}  \qquad (1)$$ 
+
+<br>
+
+
+Instead of changing the input $$\boldsymbol{b}$$, we can also add a perturbation to the matrix $$\boldsymbol{A}$$ (input) by a small amount $$\boldsymbol{E}$$, such that 
+
+$$
+\begin{align}
+(\boldsymbol{A} + \boldsymbol{E}) \hat{\boldsymbol{x}} = \boldsymbol{b}
+\end{align}
+$$
+
+and in a similar way obtain:
+
+$$\frac{\|\Delta \boldsymbol{x}\|}{\|\boldsymbol{x}\|} \le \text{cond}({\bf A})\frac{\| \boldsymbol{E}\|}{\|\boldsymbol{A}\|}  $$
 
 
 Therefore, if we know the relative error in input, then we can use the condition number of the system to obtain an upper bound for the relative error of our computed solution (output).
 
+
+#### Examples
+
+PLACEHOLDER
+
+## Condition Number
+
+#### Condition Number Definition
+
+**_Definition_**: a measure of sensitivity of solving a linear system of equations to variations in the input.
+
+The **_condition number_** of a square nonsingular matrix <span>$${\bf A}$$</span> is defined by
+$$\text{cond}({\bf A}) = \kappa({\bf A}) = \|{\bf A}\| \|{\bf A}^{-1}\| $$
+which is also the condition number associated with solving the linear system $${\bf A} \boldsymbol{x} = \boldsymbol{b}$$. A matrix with a large condition number is said to be **_ill-conditioned_**, while a matrix with a small condition is said to be **_well-conditioned_**.
+
+Recall that the induced matrix norm is given by:
+
+$$
+\begin{align}
+\|{\bf A}\|_p := \max_{\|\mathbf{x}\|=1} \|{\bf A}\mathbf{x}\|_p
+\end{align}
+$$
+
+The condition number can be measured with any <span>$$p$$</span>-norm, so to be precise we typically specify the norm being used, i.e. $$\text{cond}_2$$, $$\text{cond}_1, \text{cond}_{\infty}$$.
+
+If <span>$${\bf A}$$</span> is singular ($${\bf A}^{-1}$$ does not exist), we can define $$\text{cond}({\bf A}) = \infty$$ by convention.
+
+The identity matrix is well conditioned. Assuming the inverse of $$\|{\bf A}\|$$ exists, $$\text{cond}({\bf A}) = \|{\bf A}\| \|{\bf A}^{-1}\| \geq \|{\bf A}{\bf A}^{-1}\| = \|{\bf I}\| = 1.$$ This is the smallest possible condition number. Small condition numbers correspond to little error amplification. Remember, small condition numbers are good!
```

Comment:
Second sentence doesn't logically follow from the first sentence, maybe preface with "We show by", etc.